src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -105,4 +139,11 @@ public static EntityPlayer getPlayerFromUUID(@Nullable UUID uuid)
         return null;
     }
 
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private ServerUtils()

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)